trampoline/src/test/java/org/ernest/EncryptServiceTest.java

```diff
@@ -0,0 +1,24 @@
+package org.ernest;
+
+import org.ernest.applications.trampoline.services.EncryptService;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EncryptServiceTest {
+
+    @Test
+    public void givenValueWhenEncryptAndDecryptThenSameValueIsObtained() throws Exception {
+        String value = "Hello World";
```

Comment:
How about using `UUID.randomUUID().toString()` so that this test is not biased to a constant string value?